content/concepts/consensus-network/federated-sidechains.md

```diff
@@ -0,0 +1,91 @@
+
+_Main chain_: XRP Ledger where assets originate and where assets will be locked while being used on the side chain. For most applications, the main chain will be the XRP Ledger Devnet, Testnet, or Mainnet.
+
+_Side chain_: Custom independent ledger where proxy assets for the locked main chain assets are issued. Side chains may have rules, transactors, and validators that are very different from the main chain. Proxy assets on the side chain can be sent back to the main chain where they will be unlocked from the control of the federators.
+
+_Door account_: An account controlled by the federators. There are two door accounts: one on the main chain and one on the side chain. Cross chain transactions are started by users sending assets to a door account. Main chain to side chain transactions cause the balance to increase on the main chain door account and the balance to decrease on the side chain door account. It is called a "door" because it is the mechanism to move assets from one chain to another - much like going between rooms in a house requires stepping through a door.
+
+_Triggering transaction_: A transaction that causes the federators to start the process of signing and submitting a new response transaction. For example, sending XRP to the main chain's door account is a triggering transaction that will cause the federators to submit a new transaction on the side chain.
+
+_Response transaction_: A transaction submitted by the federators in reaction to a triggering transaction. Note that triggering transaction and response transaction depends on context. Sending XRP from a door account to a user account is a response transaction when thinking about cross chain transactions. It is a triggering transaction when thinking about how to handle failed transactions.
+
+
+## How to Set Up a Federated Sidechain
+
+Federated Sidechains are currently available as an Engineering Preview so you can experiment and explore the potential of sidechains. You can connect sidechains to the XRP Ledger Testnet, Devnet, or Mainnet as soon as network servers upgrade to XRPL 1.8.
+
+**Caution:** You can connect sidechains to the XRP Ledger Mainnet to develop and test with small amounts; it is not recommended for production use cases until federated sidechains are available in a release. 
+
+Setting up a sidechain involves the following high-level steps:
+
+1. Clone the rippled sidechain branch: https://github.com/ripple/rippled/tree/sidechain.
+2. Write custom transactors for your sidechain. Note that this is an important and non-trivial task. Refer to the XRP Ledger Protocol Reference for information about transaction types.
```

Comment:
```suggestion
2. Customize the source code for your sidechain. For example, you may want to write custom [transaction types](transaction-types.html). Note that this is an important and non-trivial task.
```

- Slightly more generic and optional. Custom transactors are not a strict requirement for a sidechain after all.
- Link transaction types instead of using text to send people to the reference.